nestkernel/model_manager_impl.h

```diff
@@ -55,8 +55,10 @@ ModelManager::register_node_model( const Name& name, std::string deprecation_inf
 
 template < template < typename targetidentifierT > class ConnectionT >
 void
-ModelManager::register_connection_model( const std::string& name, const RegisterConnectionModelFlags flags )
+ModelManager::register_connection_model( const std::string& name )
 {
+  RegisterConnectionModelFlags flags = ConnectionT< TargetIdentifierPtrRport >::flags;
+
```

Comment:
Instead of getting the `flags` out of the `ConnectionT` here, and putting them into the `GenericConnectorModel` in the form of individual `bool`s in lines 64ff. and 75ff., the `GenericConnectorModel` constructor could just get them out of the `ConnectionT` by itself. The `ConnectionT` is a template argument to that after all.

The `if` in line 73 should also be changed to use the flag from the `ConnectionT` directly instead of using the "intermediate" `flags` variable.